settings/PandoraSettings_LArRecoND_ThreeD.xml

```diff
@@ -62,6 +62,7 @@
 	<MCIdLeafName>mcp_id</MCIdLeafName>
 	<MCLocalIdLeafName>mcp_idLocal</MCLocalIdLeafName>
 	<EventsToSkip>0</EventsToSkip>
+	<MinHitsToSkip>2</MinHitsToSkip>
```

Comment:
It would be good to make sure we are using the same parameters for the deep-learning vertexing xml settings file `PandoraSettings_LArRecoND_ThreeD_DLVtx.xml` as well.